docs/new-user.rst

```diff
@@ -0,0 +1,20 @@
+New Alexa developer
+==============================================
+
+If the skills development for alexa is a new thing for you, we have some suggestion to get you deep into this world.
+
+AWS Developer account
+---------------------
+
+The first thing you need to do is configure your AWS developer account, `here <http://docs.aws.amazon.com/lambda/latest/dg/setting-up.html>`_ you can see the instructions step by step
+
+
+AWS Lambda function for a custom skill
+--------------------------------------
+
+You need to create a lambda function in order to have your skill up and running, `here you can see how <https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/developing-an-alexa-skill-as-a-lambda-function>`_ 
+
+
+Registering an Alexa skill
+--------------------------
+To configure a new skill, you need an account on the AWS developer portal. If you don't already have an account, go to `aws-developer-account <https://developer.amazon.com/login.html>`_ and create an account. After that, you can follow `these instructions <https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/registering-and-managing-alexa-skills-in-the-developer-portal#registering-an-alexa-skill>`_ to register your skill
```

Comment:
you go to the Amazon developer portal, not AWS, and this is actually the first thing you'll need